app/presenters/chf/file_set_presenter.rb

```diff
@@ -20,6 +20,12 @@ def representative_file_id
       end
     end
 
+    def representative_checksum
+      # I can not explain why this didn't work with a delegate to solr_document
```

Comment:
yeah, I figured out what was going on eventually! AFter I committed so didn't make it into my comments, I should fix them. 

Man, this stack.